src/NewWindow.js

```diff
@@ -194,40 +197,87 @@ function copyStyles(source, target) {
       console.error(err)
     }
     if (rules) {
-      const newStyleEl = source.createElement('style')
+      // IE11 is very slow for appendChild, so use plain string here
+      const ruleText = []
 
       // Write the text of each rule into the body of the style element
       Array.from(styleSheet.cssRules).forEach(cssRule => {
-        const { cssText, type } = cssRule
-        let returnText = cssText
-        // Check if the cssRule type is CSSImportRule (3) or CSSFontFaceRule (5) to handle local imports on a about:blank page
-        // '/custom.css' turns to 'http://my-site.com/custom.css'
-        if ([3, 5].includes(type)) {
-          returnText = cssText
-            .split('url(')
-            .map(line => {
-              if (line[1] === '/') {
-                return `${line.slice(0, 1)}${
-                  window.location.origin
-                }${line.slice(1)}`
-              }
-              return line
-            })
-            .join('url(')
+        const { type } = cssRule
+
+        // Skip unknown rules
+        if (type === CSSRule.UNKNOWN_RULE) {
+          return
+        }
+
+        let returnText = ''
+
+        if (type === CSSRule.KEYFRAMES_RULE) {
+          // IE11 will throw error when trying to access cssText property, so we
+          // need to assemble them
+          returnText = getKeyFrameText(cssRule)
+        } else if (
+          [CSSRule.IMPORT_RULE, CSSRule.FONT_FACE_RULE].includes(type)
+        ) {
+          // Check if the cssRule type is CSSImportRule (3) or CSSFontFaceRule (5)
+          // to handle local imports on a about:blank page
+          // '/custom.css' turns to 'http://my-site.com/custom.css'
+          returnText = fixUrlForRule(cssRule)
+        } else {
+          returnText = cssRule.cssText
         }
-        newStyleEl.appendChild(source.createTextNode(returnText))
+        ruleText.push(returnText)
       })
 
-      target.head.appendChild(newStyleEl)
+      const newStyleEl = target.createElement('style')
+      newStyleEl.contentText = ruleText.join('\n')
```

Comment:
Should it be `textContent` instead of `contentText`?

Reply:
I pulled down your PR, and it should be `textContent ` indeed`